lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu.rb'
+require_relative 'order.rb'
+require_relative 'twilio.rb'
+
+class Takeaway
+
+attr_accessor :final_order
+
+  def initialize(menu=Menu.new)
+    @menu = menu
+    @order = Order.new
+    @message = Message.new
+  end
+
+  def show_menu
+    @menu.menu_list
+  end
+
+  def order(dish, number=1)
+    fail "Item not on the menu: please choose again" unless @menu.menu_list.include? dish
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)